src/test/java/api/support/builders/LoanBuilder.java

```diff
@@ -27,16 +27,17 @@
   private final DateTime systemReturnDate;
   private final UUID checkoutServicePointId;
   private final UUID checkinServicePointId;
+  private final UUID patronGroupAtCheckout;
```

Comment:
Is this still needed? I didn’t notice any usages

Reply:
Removed